django/applications/catmaid/control/annotation.py

```diff
@@ -1349,15 +1347,15 @@ def list_annotations(request:HttpRequest, project_id=None) -> JsonResponse:
 
     # Create a set mapping annotation names to its users
     ids = {}
-    annotation_dict = {} # type: Dict
+    annotation_dict:Dict[Any, List[Dict]] = {}
     for annotation, aid, uid, username in annotation_tuples:
         ids[aid] = annotation
         ls = annotation_dict.get(aid)
-        if ls is None:
+        if ls is None: # With these two conditionals, we're making sure an empty entry exists even if uid is None. Revise if unnecessary
             ls = []
             annotation_dict[aid] = ls
         if uid is not None:
-            ls.append({'id': uid, 'name': username})
+            annotation_dict[aid].append({'id': uid, 'name': username})
```

Comment:
If with this change we are not using `ls` here anymore, we might as well replace L1353 + L1354 with `if aid not  in annotation_dict:` and L1355 + L1356 with `annotation_dict[aid] = []`. Otherwise `ls` seems kind of lost.